$OpenBSD: patch-hpodder_cabal,v 1.3 2011/05/26 08:24:59 dcoppa Exp $

fix from upstream git:
remove unneeded haskell98 dependency
allow base-4
tighten HaXml dependency

--- hpodder.cabal.orig	Wed Jul  2 17:32:27 2008
+++ hpodder.cabal	Sun Dec 26 16:08:02 2010
@@ -68,8 +68,8 @@ Description:  Podcasting is a method of publishing rad
  hpodder is SAFE and is designed with data integrity in mind from the
  beginning.  It should be exceedingly difficult to lose a podcast
  episode, even in the event of a power failure.
-Build-Depends: haskell98, network, unix, parsec, MissingH>=1.0.0,
- HDBC>=1.1.0, HDBC-sqlite3>=1.1.0, mtl, base, HaXml>=1.13.2, HaXml<1.19, hslogger,
+Build-Depends: network, unix, parsec, MissingH>=1.0.0,
+ HDBC>=1.1.0, HDBC-sqlite3>=1.1.0, mtl, base>=3 && < 5, HaXml == 1.20.*, hslogger,
  ConfigFile, filepath, old-time, directory, process
 
 Executable: hpodder
